public/static/js/filters/filters.common.js

```diff
@@ -57,9 +57,11 @@ var convertSingleTest = function (test, execution) {
     }
   }
   var result = {};
-  var threads
+  var threads = undefined;
```

Comment:
this shouldn't be necessary, it will start out as undefined

Reply:
Yes, I know. I just like to be explicit for things like this. That way future viewers of the code know that the value is meant to be undefined and it just wasn't forgotten about.